core/src/main/java/org/elasticsearch/action/bulk/BulkItemResponse.java

```diff
@@ -184,6 +192,11 @@ public Failure(StreamInput in) throws IOException {
             id = in.readOptionalString();
             cause = in.readException();
             status = ExceptionsHelper.status(cause);
+            if (in.getVersion().onOrAfter(Version.V_6_0_0_alpha1_UNRELEASED)) {
+                seqNo = in.readLong();
```

Comment:
I think @jasontedor want's us to strandardize on readZlong. @jasontedor correct me if I'm wrong..